src/libstd/sys/unix/thread.rs

```diff
@@ -1,3 +1,5 @@
+// Copyright 2019 Joyent, Inc.
```

Comment:
As far as I know, we want things added to the standard library and the compiler to be licensed under the general terms as found in `{git_root}/LICENSE-{MIT,APACHE}`.

Reply:
It is. As per COPYRIGHT:

```
Copyrights in the Rust project are retained by their contributors. No
copyright assignment is required to contribute to the Rust project.

Some files include explicit copyright notices and/or license notices.
For full authorship information, see the version control history or
https://thanks.rust-lang.org

Except as otherwise noted (below and/or in individual files), Rust is
licensed under the Apache License, Version 2.0 <LICENSE-APACHE> or
<http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
<LICENSE-MIT> or <http://opensource.org/licenses/MIT>, at your option.
```